doc/dev.html

```diff
@@ -64,8 +64,26 @@
 
 <hr />
 
-<h1 id="tasks">Compiling, testing, releasing, etc.</h1>
+<h1 id="setup">Setting up</h1>
+
+Randoop development involves working with both Randoop's code base and the
+website and documentation.
+The first is is in the master (and other branches) of the repository, while
+the website is in the gh-pages branch.
+These instructions assume that you keep subdirectories for both your working
+branch, and the gh-pages branch of Randoop in the same directory.
+Assuming this directory is <code>randoop-working</code>, get started by these steps:
+<code><pre>
+  cd randoop-working
+  git clone git@github.com:randoop/randoop.git myfeature
+  git clone git@github.com:randoop/randoop.git randoop-gh-pages
+  cd randoop-gh-pages
+  git checkout gh-pages
+</pre></code>
+You would then work in the directory <code>myfeature</code>, and follow the
+directions below to move changes into <code>gh-pages</code>.
```

Comment:
There is no reference to where to find "the directions below".  (It's in the release instructions, but that will not be obvious to readers, who are going to be confused while reading this.)  In fact, the developer doesn't have to even clone the gh-pages branch because the release instructions do so automatically if the gh-pages branch doesn't already exist.  Therefore, this entire section about setup seems redundant.  Developers will create a clone (and will probably name it `randoop`, though it doesn't matter to you whether they do), and then everything else happens automatically without need for the developer to do anything more.